src/utils/datetime/validator.ts

```diff
@@ -78,23 +78,23 @@ const formatterToDateFnsMap = {
   },
 }
 
-const getDateFnsFormatString = (format: string): string => {
-  const dateFnsFormatString = formatterToDateFnsMap[format].format
-    ? formatterToDateFnsMap[format].format
-    : formatterToDateFnsMap[DEFAULT_TIME_FORMAT].format
+export const getLuxonFormatString = (format: string): string => {
+  const luxonFormatString = formatToLuxonMap[format].format
+    ? formatToLuxonMap[format].format
+    : formatToLuxonMap[DEFAULT_TIME_FORMAT].format
 
-  return dateFnsFormatString
+  return luxonFormatString
 }
 
 const strictCheck = (dateString: string, format: string): boolean => {
-  return formatterToDateFnsMap[format].regex.test(dateString)
+  return formatToLuxonMap[format].regex.test(dateString)
```

Comment:
Since we switched over from date-fns to luxon, some refactor in variable naming.